framework/src/iotbus/iotbus_uart.c

```diff
@@ -183,12 +181,20 @@ iotbus_uart_context_h iotbus_uart_init(const char *path)
 
 	handle->fd = fd;
 	handle->device = _iotbus_get_dev_number(path);
-	for (i = 0; i < CONFIG_IOTBUS_UART_EVENT_SIZE; i++) {
-		handle->evt_hnd[i] = NULL;
-	}
 	handle->callback = NULL;
+	for (i = 0; i < IOTBUS_UART_INTR_MAX; i++) {
+		handle->cb[i] = NULL;
+	}
 	dev->handle = handle;
 
+	ret = ioctl(fd, TIOCSETIOTBUS, (unsigned long)dev);
```

Comment:
TIOCSETIOTBUS is not good naming for ioctl's constant.
Driver should work alone even if user doesn't want iotbus.